### ISSUE TITLE
Use the smb protocol incorrectly

### ISSUE THREAD

**Issue by bellaperez** [mbunkus/mkvtoolnix#1479]
Hi,

The menú option "Open folder" on task into "Job queue", open the folder into Dolphin app with the smb protocol: smb://home/user/Videos/pelis/

The correct would be: /home/user/Videos/pelis/

I use:
Ubuntu Wily Werewolf (development branch)
mkvmerge v8.5.0 ('Vanishing Act') 64bit (compiled from sources)
KDE Frameworks 5.15.0
Qt 5.4.2

I do not use Samba for nothing, although it is installed (no configuration).

Regards
Toni

**Comment by mbunkus** [comment]
Interesting case. Turns out that the GUI is adding one superfluous slash at the beginning; it tells Qt to open `file:////home/mosu/…`. Two slashes are needed for the protocol, one for the root folder, and the fourth one is superfluous.

Now file managers like Windows Explorer or krusader (my primary file manager and the one I tested with on Linux) simply open a directory name called `//home/mosu/…` – that works. But Dolphin tries to be smart, sees that the file name starts with `//…` and thinks the user wants to visit a share called `mosu` on a Samba/Windows server called `home`. Hence the result you see.

I'll remove that superfluous slash. Thanks for the report.
